### PR TITLE
v0.7.0: update hubot-stackstorm to 0.12.0

### PR DESCRIPTION
**Pull request by cognifloyd**
The new hubot-stackstorm release includes https://github.com/StackStorm/hubot-stackstorm/pull/214

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
     "hubot-scripts": "^2.17.2",
     "hubot-slack": "^4.5.5",
     "hubot-spark": "git+https://github.com/tonybaloney/hubot-spark.git#c440504",
-    "hubot-stackstorm": "^0.11.3",
+    "hubot-stackstorm": "^0.12.0",
```
Comment (arm4b) [inline package.json:24]: You need to also update the [npm-shrinkwrap.json](https://github.com/StackStorm/st2chatops/blob/master/npm-shrinkwrap.json) lock file so the new version will be really installed.

Comment (arm4b) [reply package.json:24]: The documentation on how to do that properly is here: https://github.com/StackStorm/st2chatops/issues/104
Pinned the issue again as it's not the first time it's missed.

Comment (cognifloyd) [reply package.json:24]: done. Hopefully it works.

**Comment by cognifloyd** [comment]
OK. Build is passing after cleaning up the shrinkwrap file manually in https://github.com/StackStorm/st2chatops/pull/175/commits/223f6db93641b9c54c58becbe659a9c070f22649 - It was somehow picking up a `skills-validator` dep as a file dep for msteams. That's not listed in https://www.npmjs.com/package/botbuilder/v/3.30.0 dependencies so I just deleted that from npm-shrinkwrap.json.

**Review by arm4b — APPROVED** [review]

**Pull request merged**